#
# Submitted:
# 
# Robert Schwebel, 2003-12-22
#  
# Error:
# 
# ... undefined reference to mcount_internal ...
# 
# Description: 
#
# State:
# 
# unknown
#

--- glibc-2.3.2/sysdeps/arm/machine-gmon.h	2001-07-07 21:21:19.000000000 +0200
+++ glibc-2.3.2-ptx/sysdeps/arm/machine-gmon.h	2003-12-21 23:58:26.000000000 +0100
@@ -32,10 +32,8 @@
 weak_alias (_mcount, mcount)
 #endif
 
-static void mcount_internal (u_long frompc, u_long selfpc);
-
 #define _MCOUNT_DECL(frompc, selfpc) \
-static void mcount_internal (u_long frompc, u_long selfpc)
+void mcount_internal (u_long frompc, u_long selfpc)
 
 /* This macro/func MUST save r0, r1 because the compiler inserts
 	blind calls to _mount(), ignoring the fact that _mcount may
